### ISSUE TITLE
RAM usage

### ISSUE THREAD

**Issue by MuhammadUsman-coder** [jaeles-project/gospider#56]
Gospider uses a lot of RAM.
It's ram usage keeps increasing with time . And it chokes my server if it keeps running for a few hours even with low threads (2-3).

Is there any solution to this? or can you kindly solve this issue if possible?

Thanks

**Comment by shubhack-repo** [comment]
Yes i am getting this problem too on cloud vps, it would be great if you can add some flag for solving this problem ?
